packages/python-runner/unpack.sh

```diff
@@ -2,4 +2,4 @@
 
 set -e
 
-tar zxf - -C /package 2> $PACKAGE_DIR/.fail && touch /package/.ready || touch $PACKAGE_DIR/.fail
+tar zxf - -C /package 2>&2 && touch /package/.ready || touch $PACKAGE_DIR/.fail
```

Comment:
```suggestion
tar zxf - -C /package && touch /package/.ready || touch $PACKAGE_DIR/.fail
```

There's no need to redirect stderr to stderr